parsec/core/gui/parsec_application.py

```diff
@@ -13,6 +16,8 @@ def __init__(self):
         self.setOrganizationName("Scille")
         self.setOrganizationDomain("parsec.cloud")
         self.setApplicationName("Parsec")
+        path = os.path.join(os.path.dirname(sys.modules[__name__].__file__), 'parsec.png')
+        self.setWindowIcon(QIcon(path))
```

Comment:
The window icon set in the .ui file is not enough? Or is it a problem of the resource file not being compiled on Osx?

Reply:
Despite the current settings, the taskbar/dock icon was still the default Python icon. This was the only proper fix I could find to display a custom icon.

Reply:
Could you try `self.setWindowIcon(QIcon(":/logos/images/icons/parsec.png"))` instead? It would at least use the icon in the qrc file instead of using a path. It this doesn't work, then it's a problem of the resource file.

Reply:
Works fine ! I'll have to check if this addition is still necessary when we'll have Parsec in `.app` format,  this behaviour might change then.